content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- external-workspace-manager
+- gsoc
+author: alexsomai
+---
+
+Currently it's quite difficult to share and reuse the same workspace between multiple jobs and across nodes.
```

Comment:
I think the introduction paragraph should refer to the [original blog post](https://jenkins.io/blog/2016/05/23/external-workspace-manager-plugin/) for more information about the project.